packages/dm-core-plugins/src/form/templates/ObjectModelContainedTemplate.tsx

```diff
@@ -24,24 +24,43 @@ export const ObjectModelContainedTemplate = (
     uiAttribute?.showExpanded ?? config.showExpanded
   )
   const value = watch(namePath)
-  const isCreated = value && Object.keys(value).length > 0
+  const objectIsNotEmpty = value && Object.keys(value).length > 0
   const canExpand =
-    isCreated &&
+    objectIsNotEmpty &&
```

Comment:
Should this also be true if ever "canOpenInTab" is false. I think so

Reply:
EO had a pull request fixing this